site-manage/site-manage-tool/tool/src/webapp/js/site-manage.js

```diff
@@ -11,67 +11,63 @@ $.ajaxSetup({
  calling template has dom placeholder for dialog,
  args:class of trigger, id of dialog, message strings
  */
-sakai.getSiteInfo = function(trigger, dialogTarget, nosd, nold){
-	$("." + trigger).click(function(e){
-		e.preventDefault();
-		$("#" + dialogTarget).modal('show');
-	});
-	const siteId = $("." + trigger).attr('id');
-	if (!siteId) {
-		return;
-	}
-	const siteURL = `/direct/site/${siteId}/info.json`;
-	jQuery.getJSON(siteURL, function(data){
-		var desc = '', shortdesc = '', title = '', owner = '', email = '';
-		if (data.description) {
-			desc = unescape(data.description);
-		}
-		else {
-			desc = nold;
-		}
-		if (data.shortDescription) {
-			shortdesc = data.shortDescription;
-		}
-		else {
-			shortdesc = nosd;
-		}
-
-		if (data.contactName) {
-			owner = data.contactName;
-		}
-
-		if (data.contactEmail) {
-			email = " (<a href=\"mailto:" + data.contactEmail.escapeHTML() + "\" id=\"email\">" + data.contactEmail.escapeHTML() + "</a>)";
-		}
-
-		if (data.props) {
-			if (data.props['contact-name']) {
-				owner = data.props['contact-name'];
-			}
-
-			if (data.props['contact-email']) {
-				email = "(<a href=\"mailto:" + data.props['contact-email'].escapeHTML() + "\" id=\"email\">" + data.props['contact-email'].escapeHTML() + "</a>)";
-			}
-		}
-
-		sitetitle = data.title.escapeHTML();
-		content = (
-			'<div class="modal-dialog modal-md">' +
-				'<div class="modal-content">' +
-					'<div class="modal-header">' +
-						'<button type="button" class="close" data-dismiss="modal" aria-label="Close"><span class="fa fa-times" aria-hidden="true"></span></button>' +
-						'<h4 class="modal-title">' + sitetitle + '</h4>' +
-					'</div>' +
-					'<div class="modal-body">' +
-						'<p>' + shortdesc + '</p>' +
-						'<div>' + desc + '</div>' +
-					'</div>' +
-				'</div>' +
-			'</div>'
-		);
-		$("#" + dialogTarget).html(content).attr('aria-hidden','true').attr('tabindex', '-1').attr('role', 'dialog').addClass('modal fade');
-		return false;
-	});
+ sakai.getSiteInfo = function(trigger, dialogTarget, nosd, nold) {
+  
+  const siteElements = document.querySelectorAll("." + trigger);
+
+  siteElements.forEach(function(element) {
+    element.addEventListener("click", function(e) {
+      e.preventDefault();
+      const siteId = element.id;
+
+      if (!siteId) {
+        return;
```

Comment:
I would add a console.error here just to note that there should have been an id on this element